phoenix-core/src/main/java/org/apache/phoenix/exception/SQLExceptionCode.java

```diff
@@ -453,8 +453,13 @@ public SQLException newException(SQLExceptionInfo info) {
     OPERATION_TIMED_OUT(6000, "TIM01", "Operation timed out.", new Factory() {
         @Override
         public SQLException newException(SQLExceptionInfo info) {
-            return new SQLTimeoutException(OPERATION_TIMED_OUT.getMessage(),
-                    OPERATION_TIMED_OUT.getSQLState(), OPERATION_TIMED_OUT.getErrorCode());
+            final String reason =
+                (info.getMessage() != null ? info.getMessage() : "")
```

Comment:
> info.getMessage() != null ? info.getMessage() : ""

In else block, can we use OPERATION_TIMED_OUT.getMessage(). Atleast we will know it was operation timed out.